src/api/alert_words/addAlertWords.js

```diff
@@ -0,0 +1,7 @@
+/* @flow strict-local */
+import type { ApiResponse, Auth } from '../transportTypes';
+import type { AlertWordsState } from '../../types';
+import { apiPost } from '../apiFetch';
+
+export default (auth: Auth, alertWords: AlertWordsState): Promise<ApiResponse> =>
```

Comment:
`AlertWordsState` describes our Redux state, which is an internal implementation detail of the app. If we change how we're storing that state in Redux, this API call should *not* change. (And conversely, if the API were ever to change (though it shouldn't), we also wouldn't necessarily want to change our internal state to be the same type.)

This really should be exactly `string[]`.